includes/entities_rss.inc

```diff
@@ -69,22 +70,21 @@ function islandora_entities_get_rss_related($identifier, $type) {
   );
   $query = "+RELS_EXT_hasModel_uri_mt:\"$content_model\" +mods_name_personal_displayForm_mt:\"($identifier)\"";
```

Comment:
@whikloj could you swap those + signs for %2B please (no extra spaces!) or the query picks up stuff it shouldn't. Sorry I missed this in the first pass.